### PR TITLE
few more cleanups against #1458

### PR DESCRIPTION
**Pull request by aviatesk**
A few improvements against #1458 
- add `dom` library for TS compile, which will make CI pass again
- ~disable no-longer needed command URI option~ nvm: we will use this in the future for handling links, etc
- simplify the get/set documentation logic
- handle "no-editor" case

### CODE REVIEW

File: src/docbrowser/documentation.ts [hunk 1]
```diff
@@ -137,15 +138,8 @@ async function getDocumentation(): Promise<string> {
...
-    if (!inner) { return }
-    g_forwardStack = [] // initialize forward page stack for manual search
-    showDocumentationPane()
-    const html = createWebviewHTML(inner)
-    _setHTML(html)
-}
-
-function createWebviewHTML(inner: string) {
+function createWebviewHTML(docAsMD: string) {
+    const docAsHTML = md.render(docAsMD)
```
Comment (aviatesk) [inline src/docbrowser/documentation.ts:142]: Do we want to wrap this in try/catch ?

Comment (davidanthoff) [reply src/docbrowser/documentation.ts:143]: I don't think so, if there is a bug in `md.render` we would want to know about it, right? Or could this throw an error because of a malformed doc string? Is there such a thing as invalid markdown?

File: tsconfig.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
         "target": "es6",
         "outDir": "out",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
```
Comment (davidanthoff) [inline tsconfig.json:8]: Why this change?
